terraform/oracle/cam-ora/oradbaas/main.tf

```diff
@@ -0,0 +1,27 @@
+#####################################################################

```

Comment:
Should have consistent headers across TF files... and just generically licensed under Apache 2?

Reply:
You mean all tf files should have the exact same header? I can do that but wait for your "boiler plate" with Apache 2 license text - or is the LICENSE file at the top of your project sufficient and I should just remove the "IBM Copyright" ?

Reply:
Updated all tf and code files to show Apache 2.0 license